tests/functional/lifecycle/LifecycleObjectProcessor.spec.js

```diff
@@ -34,13 +34,16 @@ describe('Lifecycle Object Processor', function lifecycleObjectProcessor() {
                     return done(err);
                 }
 
-                lop._consumer.onEntryCommittable = () => {
+                const objectTaskConsumer =
+                    lop._consumers.getConsumer(lcConfig.objectTasksTopic);
```

Comment:
Not sure the best way to change this but using the `_consumers` directly feels off, but maybe it shouldn't be public so nothing we can do